pmlc/transforms/passes.td

```diff
@@ -0,0 +1,19 @@
+#ifndef __PMLC_TRANSFORM_PASSES__
+#define __PMLC_TRANSFORM_PASSES__
+
+include "mlir/Pass/PassBase.td"
+
+def MakeEntrypointsPass : Pass<"pmlc-make-entrypoints", "mlir::ModuleOp"> {
```

Comment:
It seems like this pass should operate at the llvm dialect level, so I'd call it `llvm-make-entrypoints` or perhaps some better name (can't think of one right now).